crates/std/src/uncategorized/mesh.rs

```diff
@@ -113,22 +101,18 @@ impl Mesh {
             }
         }
     }
-    /// Flips indicies so that what was front facing will become back facing and vice versa
-    pub fn invert_indicies(&mut self) {
-        if let Some(indicies) = &mut self.indices {
-            for chunk in indicies.chunks_exact_mut(3) {
-                chunk.swap(1, 2);
-            }
+    pub fn flip_winding(&mut self) {
+        for chunk in self.indices.chunks_exact_mut(3) {
+            chunk.swap(1, 2);
         }
     }
     pub fn apply_skin(&mut self, joint_matrices: &[Mat4]) {
-        if let (Some(positions), Some(weights), Some(indices)) = (
-            &mut self.positions,
-            &self.joint_weights,
-            &self.joint_indices,
-        ) {
-            for ((position, weight), index) in
-                positions.iter_mut().zip(weights.iter()).zip(indices.iter())
+        if let (Some(weights), Some(indices)) = (&self.joint_weights, &self.joint_indices) {
```

Comment:
We could use `zip_eq` here if we expect them to be of the same length.

If not, we could use `izip!`to not have nested tuples

Reply:
Yeah sure. I will have another look on a follow-up pass.